### DISCUSSION TITLE
GUI TERMİNAL

### DISCUSSION THREAD

**Discussion by Bekir59** [CosmosOS/Cosmos#2008] [Q&A]
Hello, How Do I Add a Gui Terminal to My Cosmos Project? (I'm Turkish Translation may be bad)

**Comment by reisxd** [comment]
Merhaba, https://github.com/CosmosOS/Cosmos/wiki/Somewhat-indepth-Canvas-Changes-explained ve https://github.com/CosmosOS/Cosmos/wiki/Non-Flickering-Canvas-GUI-Tutorial e bakabilirsin.
